app/api/schema/events.py

```diff
@@ -272,6 +272,13 @@ def validate_timezone(self, data, original_data):
                                         related_view_kwargs={'event_id': '<id>'},
                                         schema='StripeAuthorizationSchema',
                                         type_='stripe-authorization')
+    order_statistics = Relationship(attribute='order_statistics',
+                                    self_view='v1.event_order_statistics',
+                                    self_view_kwargs={'id': '<id>'},
+                                    related_view_kwargs={'id': '<id>'},
+                                    related_view='v1.order_statistics_event_detail',
+                                    schema='OrderStatisticsEventSchema',
```

Comment:
Because nothing except id is present in OrderStatisticsEventSchema

Reply:
@iamareebjamal You mean the fields present their in OrderStatisticsEventSchema
```
tickets = fields.Method("tickets_count")
    orders = fields.Method("orders_count")
    sales = fields.Method("sales_count")
```

these are not fetched, so how do i fetch them @iamareebjamal